### PR TITLE
add background color for browser does not support gradient

### PR DESCRIPTION
**Pull request by memexcep**
add fallback background color for home.css

### CODE REVIEW

File: css/home.css [hunk 1]
```diff
@@ -3,12 +3,16 @@ html, body {
 }
 
 .home_head_bar {
+    background-color: #474747; /*fallback for browser does not support gradient*/
     background: -webkit-linear-gradient(top, #373737, #212121);
+    background: -moz-linear-gradient(top, #373737, #212121);
+    background: -o-linear-gradient(top, #373737, #212121);
+    background: linear-gradient(top, #373737, #212121);
```
Comment (tianlunte) [inline css/home.css]: 这几个之间有什么区别？

Comment (memexcep) [reply css/home.css]: https://en.wikipedia.org/wiki/CSS_hack#Browser_prefixes
不同的浏览器，对这些还没完全成熟敲定的css属性实现方式略有差别。

**Review by tianlunte — CHANGES_REQUESTED** [review]
请➕上comment 以便后面的人查看

File: css/home.css [hunk 2]
```diff
@@ -3,12 +3,16 @@ html, body {
 }
 
 .home_head_bar {
-    background: -webkit-linear-gradient(top, #373737, #212121);
+    background-color: #474747; /*fallback for browser does not support gradient*/
+    background: -webkit-linear-gradient(top, #373737, #212121);／＊Each of the most popular browser rendering engines has its own vendor-specific prefix for experimental properties.*/
```
Comment (tianlunte) [inline css/home.css]: the comments should be more specific, like -o-linear-gradient is for Opera 11.1 to 12.0, we should put comment for each of these variants

**Review by nelzomal — APPROVED** [review]

**Review by nelzomal — DISMISSED** [review]

**Review by tianlunte — APPROVED** [review]

**Pull request merged**